### ISSUE TITLE
Problems may arise when processing long time series trajectories.

### ISSUE THREAD

**Issue by hutao568** [PJLab-ADG/DetZero#11]
https://github.com/PJLab-ADG/DetZero/blob/main/tracking/detzero_track/models/tracking_modules/kalman_filter/kalman_filter.py#L99

Data will overflow

**Comment by superkoma** [comment]
Yeah, we didn't update the value of Q in the following update step. For the sequence length of 200 of Waymo, it is OK to run with this Q. But if you want to run it on very long sequences, it may cause the data overflow. We may fix it in the following days.

**Comment by SISTMrL** [comment]
> Yeah, we didn't update the value of Q in the following update step. For the sequence length of 200 of Waymo, it is OK to run with this Q. But if you want to run it on very long sequences, it may cause the data overflow. We may fix it in the following days.

hello, i also encounter this problem, and looking forward to your updated version
